spec/supports/helpers.rb

```diff
@@ -0,0 +1,20 @@
+require 'aws-sdk'
+
+def purge_queue
+  # Ideally we would just call `@sqs.purge_queue(:queue_url => @queue_url)`,
+  # but this method can only be called once every 60 seconds.
+  receive_all_messages
+end
+
+def receive_all_messages
+  messages = []
+  poller = Aws::SQS::QueuePoller.new(@queue_url)
```

Comment:
Personally this seems really weird to me, but I (once again) copied the idea from `logstash-output-sqs`. Basically, it's strange to me that `@queue_url` works here... I would've expected that it would have needed to be passed in as an argument to the function.